src/redfish_collector/core/rawCollector.py

```
@@ -179,27 +184,35 @@ async def dataCollector(serverAddress,username,password,templateDir,logLevel):
-                            pass
-                        else:
-                            logging.error("[%s] Can't get X-Auth-Token from response headers" % serverAddress)
-                            return
-                    pass
-            except Exception as e:
-                logging.error("[%s] There is error when getting token: %s" % (serverAddress,e))
-                return
+                                logging.error("[%s] Can't get X-Auth-Token from response headers" % serverAddress)
+                        pass
+                    break
+                except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)